ButterCMS/ButterCMSClient.cs

```diff
@@ -49,7 +50,7 @@ private string authTokenParam
         }
         private JsonSerializerSettings serializerSettings;
 
-        public ButterCMSClient(string authToken, TimeSpan? timeOut = null, int maxRequestTries = 3, HttpMessageHandler httpMessageHandler = null)
+        public ButterCMSClient(string authToken, bool preview = false, TimeSpan? timeOut = null, int maxRequestTries = 3, HttpMessageHandler httpMessageHandler = null)
```

Comment:
@vlnevyhosteny this is backwards compatible meaning `new ButterCMSClient("API KEY");` will still work for customers correct?

What about customers who are initializing using other parameters. i.e. `new ButterCMSClient("API KEY", maxRequestTries=5);`

Would it be best to add this new `preview` parameter at the end? In case someone is doing implicit passing of params `new ButterCMSClient("API KEY", timeout, 5)`?

Reply:
@jakelumetta Yeah, you are right. Fixed in https://github.com/ButterCMS/buttercms-csharp/pull/30/commits/2728262dc22c474882d83282e50ece0c1fc04c10